kubeconfig-creator/README.md

```diff
@@ -0,0 +1,155 @@
+# Kubeconfig Creator Task
+
+
+This `Task` do a similar job to the [Cluster](https://github.com/tektoncd/pipeline/blob/master/docs/resources.md#cluster-resource) `PipelineResource` and
+are intended as its replacement. This is part of our plan to [offer replacement
+`tasks` for Pipeline Resources](https://github.com/tektoncd/catalog/issues/95)
+as well as
+[document those replacements](https://github.com/tektoncd/pipeline/issues/1369).
+
+
+This `task` creates a [kubeconfig](https://kubernetes.io/docs/tasks/access-application-cluster/configure-access-multiple-clusters/)
+file that can be used to configure access to the different clusters.
+A common use case for this task is to deploy your `application/function` on different clusters.
+
+The task will use the [kubeconfigwriter](https://github.com/Divyansh42/kubeconfigwriter) image and the provided parameters to create a `kubeconfig` file that can be used by other tasks
```

Comment:
suggest updating link to pipeline's kubeconfigwriter.

Reply:
Okay, I missed this one.